src/addressResolvers/utils.ts

```diff
@@ -34,9 +44,13 @@ export function graphQlCall(url, query: string) {
 export function normalizeAddresses(addresses: Address[]): Address[] {
```

Comment:
This has same function name as all resolvers `normalizeAddresses` could we change to something else?

Reply:
Actually we are not using this anywhere else. maybe we can just move this to addressResolvers/index.ts

Reply:
We could move it to index, but also will require importing 2 new dependencies inside `index.ts` inside of just one

```
import { getAddress } from '@ethersproject/address';
import { isStarknetAddress } from './utils';
```